src/ontology/cl-edit.owl

```diff
@@ -4023,9 +4023,12 @@ SubClassOf(obo:CL_0000117 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_0000031))
 
 # Class: obo:CL_0000118 (basket cell)
 
+AnnotationAssertion(<http://geneontology.org/formats/oboInOwl#created_by> obo:CL_0000118 "https://orcid.org/0000-0002-2825-0621")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Basket_cell&oldid=951703880") obo:IAO_0000115 obo:CL_0000118 "Basket cells are inhibitory GABAergic interneurons of the brain. In general, dendrites of basket cells are free branching and contain smooth spines. Axons are highly branched. The branched axonal arborizations give rise to the name as they appear as baskets surrounding the soma of the target cell. Basket cells form axo-somatic synapses, meaning their synapses target somas of other cells.")
```

Comment:
> The branched axonal arborizations give rise to the name as they appear as baskets surrounding the soma of the target cell.

Quick question: Is it standard to include etymology of labels in definitions? If it's not standard, then maybe "The branched axonal arborizations give rise to basket-like structures that surround the soma of the target cell."

Reply:
> Quick question: Is it standard to include etymology of labels in definitions?

I'm not too sure, I've tended to avoid explicitly stating etymology but have stated it in some way if that makes sense? (eg for chandelier cells "recognizable by the straight terminal axonal 'cartridges' of vertically oriented strings of synaptic boutons"). Would actually like to know the answer to that question though.

Anyway regardless of the answer, I think that that change is good - makes it clearer than before, will make it :) thanks!